src/main/jbake/content/documentation/the-sling-engine/service-authentication.md

```diff
@@ -100,15 +100,23 @@ described above consists of three parts:
 The first part is a new OSGi Service `ServiceUserMapper`. The
 `ServiceUserMapper` service allows for mapping *Service IDs* comprised of
 the *Service Names* defined by the providing bundles and optional *Subservice Name*
-to ResourceResolver and/or JCR Repository user IDs or principals ([SLING-6939](https://issues.apache.org/jira/browse/SLING-6963)). This mapping is configurable
+to ResourceResolver and/or JCR Repository principal names ([SLING-6939](https://issues.apache.org/jira/browse/SLING-6963)) or 
+user IDs ([SLING-6939](https://issues.apache.org/jira/browse/SLING-6963)). This mapping is configurable
 such that system administrators are in full control of assigning users to services.
 
 The `ServiceUserMapper` defines the following API:
 
     #!java
+    Iterable<String> getServicePrincipalNames(Bundle bundle, String subServiceName);
+     
+The alternative API (getting service user ID as shown below) has been deprecated for security reasons and will be removed
+in future releases. See [SLING-6939](https://issues.apache.org/jira/browse/SLING-6963) for details.
```

Comment:
6939 and link points to 6963. Which ticket tracks the deprecation? https://issues.apache.org/jira/browse/SLING-6963 is only about the introduction of the new API.

Reply:
oh.... thanks for spotting... that's obviously an oversight. the deprecation is covered with SLING-10321. i will fix it.

Reply:
Still the other link title says `SLING-6939` while the link actually points to `https://issues.apache.org/jira/browse/SLING-6963`

Reply:
@kwin , well... that's an old mistake :-) but i can fix it as well...